src/Controls/src/Core/CommandElement.cs

```diff
@@ -0,0 +1,53 @@
+﻿#nullable enable
+using System;
+using System.Windows.Input;
+using Microsoft.Maui.Controls.Internals;
+
+namespace Microsoft.Maui.Controls
+{
+	static class CommandElement
+	{
+		static readonly BindableProperty CanExecuteProperty = BindableProperty.CreateAttached(
+			"CanExecute", typeof(bool), typeof(ICommandElement), true);
```

Comment:
This probably does not need to be a bindable property? We can just let each control hande it the way they want.